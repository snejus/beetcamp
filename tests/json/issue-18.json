{
  "@context": "https://schema.org",
  "@id": "https://54thregiment.bandcamp.com/album/fall-in",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 1474409418
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 2
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://54thregiment.bandcamp.com/album/fall-in",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 3640739659
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 732004402
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 1474409418
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a1474409418_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "FALL IN"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/54-regi",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/54-regi-2",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/mile-high-monsta-ft-nation",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/pure-entertainment",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/smoke-interlude",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/ganja-tune-ft-spellbinder",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/war-ft-spellbinder",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/w-m-m-a-where-my-money-ft-spellbinder",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/due-time",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/hustle-aint-no-game",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/its-ova-ft-scrilla-scratch",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://54thregiment.bandcamp.com/track/dont-stop-ft-spellbinder",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@type": "MusicGroup",
    "name": "General Dicon"
  },
  "copyrightNotice": "All Rights Reserved",
  "dateModified": "16 May 2011 03:29:03 GMT",
  "datePublished": "20 Apr 2011 00:00:00 GMT",
  "image": "https://f4.bcbits.com/img/a1474409418_10.jpg",
  "keywords": [
    "Hip-hop",
    "Real nigga shit",
    "rap",
    "Denver"
  ],
  "mainEntityOfPage": "https://54thregiment.bandcamp.com/album/fall-in",
  "name": "FALL IN",
  "numTracks": 12,
  "publisher": {
    "@id": "https://54thregiment.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 732004402
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      }
    ],
    "foundingLocation": {
      "@type": "Place",
      "name": "Denver, Colorado"
    },
    "name": "54th Regiment",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "music",
        "url": "https://54thregiment.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://54thregiment.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/54-regi",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2788523249
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "General Dicon"
          },
          "copyrightNotice": "All Rights Reserved",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/54-regi",
          "name": "54 regi"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/54-regi-2",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1940656242
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H03M47S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/54-regi-2",
          "name": "54 regi"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/mile-high-monsta-ft-nation",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2017700002
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H02M37S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/mile-high-monsta-ft-nation",
          "name": "Mile high monsta ft.nation"
        },
        "position": 3
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/pure-entertainment",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2504455144
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "General Dicon"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H02M46S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/pure-entertainment",
          "name": "Pure entertainment"
        },
        "position": 4
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/smoke-interlude",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2191604344
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "General Dicon"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H01M09S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/smoke-interlude",
          "name": "Smoke interlude"
        },
        "position": 5
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/ganja-tune-ft-spellbinder",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3808316298
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H03M55S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/ganja-tune-ft-spellbinder",
          "name": "Ganja tune ft.spellbinder"
        },
        "position": 6
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/war-ft-spellbinder",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1546501549
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H03M23S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/war-ft-spellbinder",
          "name": "War ft.spellbinder"
        },
        "position": 7
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/w-m-m-a-where-my-money-ft-spellbinder",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3184302987
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H02M27S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/w-m-m-a-where-my-money-ft-spellbinder",
          "name": "w.m.m.a(where my money @) ft.Spellbinder"
        },
        "position": 8
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/due-time",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 524055908
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "General Dicon"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M17S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/due-time",
          "name": "Due time"
        },
        "position": 9
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/hustle-aint-no-game",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2084081972
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "General Dicon"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H03M17S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/hustle-aint-no-game",
          "name": "Hustle Ain't no game"
        },
        "position": 10
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/its-ova-ft-scrilla-scratch",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3829944235
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H03M21S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/its-ova-ft-scrilla-scratch",
          "name": "It's ova ft. scrilla scratch"
        },
        "position": 11
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://54thregiment.bandcamp.com/track/dont-stop-ft-spellbinder",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3094837483
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "General Dicon"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M04S",
          "mainEntityOfPage": "https://54thregiment.bandcamp.com/track/dont-stop-ft-spellbinder",
          "name": "Don't stop ft.Spellbinder"
        },
        "position": 12
      }
    ],
    "numberOfItems": 12
  }
}
