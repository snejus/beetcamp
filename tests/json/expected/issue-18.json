[
  {
    "album": "FALL IN",
    "album_id": "https://54thregiment.bandcamp.com/album/fall-in",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "album",
    "albumtypes": "album",
    "artist": "General Dicon",
    "artist_credit": null,
    "artist_id": "https://54thregiment.bandcamp.com",
    "artist_sort": null,
    "asin": null,
    "catalognum": "",
    "comments": null,
    "country": "US",
    "data_source": "bandcamp",
    "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
    "day": 20,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": null,
    "label": "54th Regiment",
    "language": null,
    "media": "Digital Media",
    "mediums": 1,
    "month": 4,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": null,
    "tracks": [
      {
        "arranger": null,
        "artist": "General Dicon",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": null,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 12,
        "release_track_id": null,
        "title": "54 regi",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/54-regi",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 227,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 12,
        "release_track_id": null,
        "title": "54 regi",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/54-regi-2",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon ft.nation",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 157,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 12,
        "release_track_id": null,
        "title": "Mile high monsta",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/mile-high-monsta-ft-nation",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 166,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 12,
        "release_track_id": null,
        "title": "Pure entertainment",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/pure-entertainment",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 5,
        "initial_key": null,
        "length": 69,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 5,
        "medium_total": 12,
        "release_track_id": null,
        "title": "Smoke interlude",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/smoke-interlude",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon ft.spellbinder",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 6,
        "initial_key": null,
        "length": 235,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 6,
        "medium_total": 12,
        "release_track_id": null,
        "title": "Ganja tune",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/ganja-tune-ft-spellbinder",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon ft.spellbinder",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 7,
        "initial_key": null,
        "length": 203,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 7,
        "medium_total": 12,
        "release_track_id": null,
        "title": "War",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/war-ft-spellbinder",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon ft.Spellbinder",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 8,
        "initial_key": null,
        "length": 147,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 8,
        "medium_total": 12,
        "release_track_id": null,
        "title": "w.m.m.a(where my money @)",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/w-m-m-a-where-my-money-ft-spellbinder",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 9,
        "initial_key": null,
        "length": 257,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 9,
        "medium_total": 12,
        "release_track_id": null,
        "title": "Due time",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/due-time",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 10,
        "initial_key": null,
        "length": 197,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 10,
        "medium_total": 12,
        "release_track_id": null,
        "title": "Hustle Ain't no game",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/hustle-aint-no-game",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon ft. scrilla scratch",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 11,
        "initial_key": null,
        "length": 201,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 11,
        "medium_total": 12,
        "release_track_id": null,
        "title": "It's ova",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/its-ova-ft-scrilla-scratch",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "General Dicon ft.Spellbinder",
        "artist_credit": null,
        "artist_id": "https://54thregiment.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://54thregiment.bandcamp.com/album/fall-in",
        "disctitle": null,
        "genre": null,
        "index": 12,
        "initial_key": null,
        "length": 244,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 12,
        "medium_total": 12,
        "release_track_id": null,
        "title": "Don't stop",
        "track_alt": null,
        "track_id": "https://54thregiment.bandcamp.com/track/dont-stop-ft-spellbinder",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2011
  }
]
